packages/main/cypress/specs/Panel.cy.tsx

```diff
@@ -0,0 +1,474 @@
+
+		cy.get("@header")
+			.realPress("Space");
+		
+		// eslint-disable-next-line cypress/no-unnecessary-waiting
+		cy.wait(50);
+
+		cy.get("@content")
+			.should("be.visible");
+
+		cy.get("@header")
+			.should("be.focused");
+
+		cy.get("@header")
+			.realPress("Enter");
+
+		// eslint-disable-next-line cypress/no-unnecessary-waiting
+		cy.wait(50);
+
+		cy.get("@content")
+			.should("not.be.visible");
+	});
+
+	it("Sticky header when scrolling inner content", () => {
+		cy.document().then((doc) => {
+			const style = doc.createElement('style');
+			style.id = "panel-stickyHeader-style";
+			style.innerHTML = `
+				#panel-stickyHeader::part(content) {
+					max-height: 70px;
+				}
+			`;
+			doc.head.appendChild(style);
+		  });
+		cy.mount(<Panel headerText="Fixed Panel" stickyHeader={true} id="panel-stickyHeader">
+			<Label wrappingType="Normal">Lorem ipsum dolor sit amet, tamquam invidunt cu sed, unum regione mel ea, quo ea alia novum. Ne qui illud zril
+				nostrum, vel ea sint dicant postea. Vel ne facete tritani, neglegentur concludaturque sed te. His animal dolorum ut.
+				Aeterno appareat ei mei, cu sed elit scripserit, an quodsi oportere accusamus quo. Pri ea probo corpora rationibus,
+				soluta incorrupte ex his.
+				Mei ei brute cetero, id duo magna aeque torquatos. Quodsi erroribus mediocritatem his ut, ad pri legere iracundia
+				democritum. Menandri intellegam in mea, ex vero movet qualisque sed. Maiorum verterem perfecto nec ea, est velit
+				elaboraret consequuntur eu, eam ad reque postea admodum. Ne inimicus convenire pri, doctus vidisse te ius.
+				Percipitur contentiones in vis, cu vim propriae phaedrum. Has ad magna errem honestatis, duo vero graeco epicurei
+				no, populo semper sit ne. Vulputate dissentiunt interpretaris ea vis, nec civibus moderatius at. Cu vim stet
+				dissentias, no vidit saperet indoctum nec, et pro magna prima nobis. Vis consul feugiat qualisque in, regione
+				persecuti cotidieque id eos, id ius omnesque vituperata.
+			</Label>
+		</Panel>);
+
+		cy.get("[ui5-panel]")
+			.shadow()
+			.find(".ui5-panel-content")
+			.as("content");
+
+		cy.get("[ui5-panel]")
+			.shadow()
+			.find(".ui5-panel-header")
+			.as("header");
+
+		cy.get("@header")
+			.then(($headerBefore) => {
+				const headerTopBefore = $headerBefore[0].getBoundingClientRect().top;
+	
+				cy.get("@content")
+					.scrollTo("bottom");
+	
+				cy.get("@header")
+					.should(($headerAfter) => {
+						const headerTopAfter = $headerAfter[0].getBoundingClientRect().top;
+						expect(headerTopAfter).to.eq(headerTopBefore);
+					});
+			});
+
+		// Clean up the style attribute
+		cy.window()
+			.then($el => {
+				const styleTag = $el.document.head.querySelector("style[id='panel-stickyHeader-style']");
+
+				styleTag?.remove();
+			})
+	});
+
+	it("Sticky header when scrolling outer container", () => {
+		cy.document().then((doc) => {
+			const style = doc.createElement('style');
+			style.innerHTML = `
```

Comment:
clean up the styles here as well. You can extract this into function (you can use commands in functions as well and they will be attached to the cypress commands queue).

```ts
addPageStyles(styles: string)
clearPageStyles()
```